PacketFabric/terraform-provider-packetfabric
# Pfacctests/packetfabric point to point

## Description



- ACCTest for packetfabric_point_to_point
- Add disable port in dev before delete
- Add port_circuit_id as computed field in packetfabric_point_to_point
- Refactor status check for packetfabric_point_to_point
- Update checkCloudRouterConnectionStatus() -- var name and error message
- Rename PTP data-source
- Add more debugs logs in client.go if enabled
- cleanup and re-org
